### PR TITLE
Attachments - Policy and Service updates for Group and Project attachments

### PR DESCRIPTION
**Pull request by ChrisHuynh333**
## What does this PR do and why?
This policy updates the policies and `attachments` `create` service for adding attachments to `groups` and `projects`

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
1. Migrate your database
2. Verify tests pass locally

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/policies/group_policy.rb
Comment (ericenns) [inline app/policies/group_policy.rb:1]: **nitpick (blocking):** Could you please add tests for these new policies, you can find the existing test file under `test/policies/`

File: app/policies/project_policy.rb
Comment (ericenns) [inline app/policies/project_policy.rb:1]: **nitpick (blocking):** Could you please add tests for these new policies, you can find the existing test file under `test/policies/`

**Review by ericenns — CHANGES_REQUESTED** [review]
Looking good, just need a few more tests as specified in the comments. But in addition tests for the attachments destroy service are needed.

File: app/policies/group_policy.rb
Comment (ChrisHuynh333) [reply app/policies/group_policy.rb:1]: Updated in b50790012da4c3b475e8e268faa8bc838cc4e073

File: app/policies/project_policy.rb
Comment (ChrisHuynh333) [reply app/policies/project_policy.rb:1]: Updated in b50790012da4c3b475e8e268faa8bc838cc4e073

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Review by ericenns — APPROVED** [review]

**Pull request merged**